hydra-cluster/test/Test/GeneratorSpec.hs

```diff
@@ -71,3 +72,27 @@ apply globals ledgerEnv utxo tx =
 
 prettyJSONString :: ToJSON a => a -> String
 prettyJSONString = unpack . decodeUtf8 . prettyPrintJSON
+
+-- TODO: this exists in the hydra-cardano package already and should be unified
+defaultGlobals :: Ledger.Globals
```

Comment:
There is really no need to duplicate the fixture here. Either make it available or just stick with loading the genesis-shelley.json like before?

Reply:
My preference: Use the `config/devnet/genesis-shelley.json` just before